c/pthread-memsafety/list2_false-valid-deref.i

```diff
@@ -1070,8 +1070,8 @@ int main(int argc, char **argv) {
   pthread_t id1, id2;
   struct item *list = ((void *)0);
   pthread_create(&id1, ((void *)0), build, list);
-  pthread_create(&id2, ((void *)0), delete, list);
```

Comment:
This change removes the main part of the concurrent program.
The program should have concurrent access to a list and then dereference some already deleted element.